scripts/checkForDuplicates/checkForDuplicates.js

```diff
@@ -17,8 +17,9 @@ fs = require('fs');
             haveDuplicates.push(akaRecord)
         }
     };
-    var file = fs.createWriteStream('DuplicatePersons.txt');
+    const file = fs.createWriteStream('DuplicatePersons.txt');
     // file.on('error', function(err) { /* error handling */ });
     file.write(JSON.stringify(haveDuplicates))
+    file.write('\r\n');
     file.end();
     })();
```

Comment:
empty row at each of source file, not of the data file of duplicates
Can be indetified by the icon 
![image](https://user-images.githubusercontent.com/62954509/99368107-73b66880-28c3-11eb-9eef-02e0e7b76376.png)